libr/core/cmd_info.c

```diff
@@ -143,18 +143,16 @@ static void r_core_file_info(RCore *core, int mode) {
 			dbg = R_IO_WRITE | R_IO_EXEC;
 		}
 		if (cf->desc) {
-			ut64 fsz = r_io_desc_size (core->io, cf->desc);
+			ut64 fsz = r_io_desc_size (cf->desc);
```

Comment:
the reason for having RIO as first arg is for consistency in RIOBind, because all other methods take RIO as first argument